descriptors/ModuleDescriptor-template.json

```diff
@@ -11,6 +11,26 @@
           "methods": ["GET"],
           "pathPattern": "/custom-fields",
           "permissionsRequired": ["custom.fields.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/custom-fields",
+          "permissionsRequired": ["custom.fields.item.post"]
```

Comment:
Shouldn't this be collection.post?

Reply:
it seems to be common to have permission named item.post
https://github.com/search?q=org%3Afolio-org+item.post+filename%3AModuleDescriptor-template.json&type=Code